app/controllers/memberships_controller.rb

```diff
@@ -0,0 +1,38 @@
+class MembershipsController < ApplicationController
+
+  def join
+    @organization = Organization.find membership_params[:id]
+    membership = @organization.memberships.new(user: current_user)
+
+    if membership.save
+      flash[:success] = "Membership request sent."
+    else
+      flash[:danger] = membership.errors.full_messages
+    end
+
+    redirect_to @organization
+  end
+
+  def deactivate
+    @membership = Membership.find(membership_params[:membership_id])
+    @membership.deactivated.save
+    redirect_to current_user
+  end
+
+  def activate
+    @membership = Membership.new(user: current_user)
+
+    if @membership.save
```

Comment:
This method doesn't appear to be activating the membership. Is there a test that covers this controller action, be it a feature test or a controller test?